README.md

```diff
@@ -27,8 +27,7 @@ Change Log
         * Prints the commands submitted to the cluster to `STDOUT` when launched.
     * New `--runfile_prefix` option to help avoid potential filename clashes.
     * Added compatability with GRIDEngine `~/.sge_request files` (by ignoring them). Thanks to [@s-andrews](https://github.com/s-andrews)
-    * New tophat module called `tophat` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).
-        * The previous tophat module is still available if you're not interested in MAPQ scores and would like slightly faster processing. This is now called `tophat_broken_MAPQ.cfmod`.
+    * New tophat module called `tophat_fixedmapq` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).
```

Comment:
Yes it is - the text you have here was the original that I replaced in https://github.com/ewels/clusterflow/commit/b60053a1fa60f69e2445fd9de8297ef75f04899e

Ok cool, I'll just change it in yours to resolve the conflict.